### PR TITLE
fix Request Ending

### PR DESCRIPTION
**Pull request by Digital39999**
Type: Bugfix
Description: When using Express which has 404/error handlers, Twurple will just continue the request even if it ended.

### CODE REVIEW

**Review by d-fischer — CHANGES_REQUESTED** [review]

File: packages/eventsub-http/src/EventSubHttpBase.ts [hunk 1]
```diff
@@ -208,6 +208,7 @@ export abstract class EventSubHttpBase extends EventSubBase {
 				res.writeHead(200, undefined);
 				res.end(verificationBody.challenge);
 				this._logger.debug(`Successfully subscribed to event: ${id}`);
+				return;
```
[the remark this answers is not in the document]
Comment (Digital39999) [reply packages/eventsub-http/src/EventSubHttpBase.ts:211]: Pretty much yes, those returns are there for code equality.

**Comment by Digital39999** [comment]
@d-fischer what now about this pr, can you merge it, or do own commit, we'd really need this in release versiom.

**Comment by d-fischer** [comment]
You resolved the conversation without actually changing anything. Additionally, the checks fail.

**Comment by Digital39999** [comment]
But there's nothing to change in conversation? I removed the next(), and added returns to make code equal as in other places, ei. [1.](https://github.com/twurple/twurple/blob/main/packages/eventsub-http/src/EventSubHttpBase.ts#L157), [2.](https://github.com/twurple/twurple/blob/main/packages/eventsub-http/src/EventSubHttpBase.ts#L171), [3.](https://github.com/twurple/twurple/blob/main/packages/eventsub-http/src/EventSubHttpBase.ts#L196), and so on..

**Comment by Digital39999** [comment]
And this package is hell to use prettier with, literally all files are marked with warns.

**Comment by d-fischer** [comment]
> And this package is hell to use prettier with, literally all files are marked with warns.

This package has its own prettier config and even runs prettier on every single commit and succeeds. The only way this could be true is if you are using a config that is not our config, which is probably your own problem.

**Comment by d-fischer** [comment]
> But there's nothing to change in conversation?

If I tell you something is useless, and even CI tells you it's useless, arguing against it is not gonna help at all.

**Comment by d-fischer** [comment]
After the last force push, it doesn't seem like there is any change in this PR anymore, other than faulty version bumps. I'm closing this, you may open a new PR if you want to move this issue forward and get credit for it. Otherwise, I'm just gonna properly fix it myself when I have the time.

**Comment by Digital39999** [comment]
I just synced it with origin and wanted to fix it in branch you redirected me to. I'll open a new issue.

**Pull request closed without merging**